CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 1.1.6
+* Added setTouchEnabled() to enable/disable zoom, pan and drag
```

Comment:
you can leave this part out -- we'll include it once we push an updated version to maven central, which may include additional changes